README.rst

```diff
@@ -74,23 +74,37 @@ Settings
 g:coverage_script
     Default: "" (which means autodetect)
 
-    Name of the script that can produce reports.  Example ::
+    Name of the script that can produce reports. Example: ::
```

Comment:
The way ReStructuredText works this can be condensed into 
```suggestion
    Name of the script that can produce reports. Example::
```

Reply:
I meant to add that colon so as not to leave a bare "Example" at the end of the line, don't really have to if you don't want.

Reply:
In ReStructuredText, colon colon starts a preformatted text block with a visible colon, while space colon colon starts a preformatted text block _without_ a visible colon.

So to make the colon visible, all you need to do is remove the space in front of `::`.

Reply:
Oh nice nice ! You can change all three of them back then, sorry for the mistake. :)